debian/postinst

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -eu
+
+g_s_helper_user=g-s-helper
+g_s_helper_home=/var/lib/${g_s_helper_user}
```

Comment:
I have chosen it like this because it's more future proof. I will move the download/build of the apps into the helper user's home but I think the repo should be a different dir. That way we can use the helper's home for other things not external apps related.